Backend/app/rider-platform/rider-app/Main/src/API/UI/Maps.hs

```diff
@@ -48,13 +48,18 @@ type API =
              :> TokenAuth
              :> ReqBody '[JSON] DMaps.GetPlaceNameReq
              :> Post '[JSON] DMaps.GetPlaceNameResp
+           :<|> "getReverseGeocode"
+             :> TokenAuth
```

Comment:
I don't know if we really need some endpoint for manual testing in prod, but if we do - it should be AdminTokenAuth

Reply:
I deleted this endpoint. I added it just to check that it was working properly.